src/index.html

```diff
@@ -6,8 +6,285 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>John Roberts' CV</title>
     <link rel="stylesheet" href="./css/index.scss" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
+    />
   </head>
+
   <body>
+    <!-- Header Section-->
+    <div class="header">
+      <a href="./index.html">Home</a>
+      <a href="">DOWNLOAD MY FULL RESUME</a>
+    </div>
+
+    <!-- Head Section-->
+    <div class="head">
+      <h1>John Roberts</h1>
+      <h2>BUSINESS EXPERT</h2>
+      <img src="" alt="" />
+    </div>
+
+    <!-- Slider show-->
+    <div class="slider-section">
+      <!-- AboutMe Slider-->
+      <div class="slider">
+        <div class="top">
+          <div class="title">
+            <h2>ABOUT ME</h2>
+          </div>
+          <div class="media_unique">
+            <a href=""><i class="fab fa-linkedin-in"></i></a>
+            <a href=""><i class="fab fa-twitter"></i></a>
+            <a href=""><i class="fab fa-facebook-f"></i></a>
+          </div>
+        </div>
+        <div class="bottom">
+          <div class="bottom-content">
```

Comment:
The `div.bottom-content` tag is redundant, since all the stylings can be assigned to its parent `div.bottom`.

For the rendering performance and the page size, we'd better to reduce all the unnecessary HTML hierarchy.